recipes/luqum/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "luqum" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.linux-x86_64.tar.gz
```

I think you are better off using the github link.

```suggestion
  url: https://github.com/jurismarches/luqum/archive/refs/tags/{{ version }}.tar.gz
```

---

Made this choice and hope it works 🤞